test/parallel/test-timers-unref-throw-then-ref.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+process.once('uncaughtException', common.expectsError({
+  message: 'Timeout Error'
+}));
+
+let called = false;
```

Comment:
`common.mustCall()` doesn't work here? It checks for one and only one call by default.

Reply:
It can potentially cause an infinite loop since it only checks at `exit` and this tests the fact that calling `ref` on `unref'd` handle doesn't cause the C++ loop to continue forever.

This is related to the `owner` check in C++. Someone could, in the future, think to optimize that by checking whether the handle is `ref` or `unref`, instead of checking for presence of an `owner`, but that would actually be incorrect and would cause an infinite loop on this test.